panc/src/test/pan/Functionality/exists/exists9.pan

```diff
@@ -10,12 +10,12 @@ object template exists9;
 
 # In particular, this returned false for v7 and returns true
 # for v8.
-variable X ?= exists(X);
+variable X = exists(X);
 
 variable Y = undef;
-variable Y ?= exists(Y);
+variable Y = exists(Y);
```

Comment:
Actually thinking about this, should the ? be removed? In all cases the behaviour should be unchanged by the code change?

Reply:
this unittest tests `exists`, not default assignment.  at the very least the current code does what the unittest describes to do (eg no failures due to issues with `?=`, but only with `exists`).
i don't mind adding more unittests for default assigment, please make a suggestion what should be tested

Reply:
But why is the change necessary? The commit message just says "panc: exists9 unittest: do not use default assigments to set the test…
… values." it doesn't say why you need to make the change. Originally I assumed it was because the code change necessitated it but from reading the code change it doesn't appear so. I think @jouvin also assumed that changing the unit test implied a behavioural code change and so he has concerns. Whereas I think this is un-connected?

Reply:
it is necessary for the variable Z exists line, since this PR changes the behaviour of default assigment when null valu is present. but this test shouldn't have used `?=` at all. i fail to see what value it adds, so i cleaned it up.

Reply:
I thought the test was about ?= but given the comment at the top of the file your interpretation makes sense.  I guess the ?= behaviour is tested in the test default24.